theano/tensor/nnet/__init__.py

```diff
@@ -151,3 +152,86 @@ def conv2d(input, filters, input_shape=None, filter_shape=None,
     return abstract_conv2d(input, filters, input_shape, filter_shape,
                            border_mode, subsample, filter_flip,
                            filter_dilation)
+
+
+def conv2d_transpose(input, filters, output_shape, filter_shape=None,
+                     border_mode='valid', input_dilation=(1, 1),
+                     filter_flip=True, filter_dilation=(1, 1)):
+    """
+    This function will build the symbolic graph for applying a transposed
+    convolution over a mini-batch of a stack of 2D inputs with a set of 2D
+    filters.
+
+
+    Parameters
+    ----------
+    input: symbolic 4D tensor
+        Mini-batch of feature map stacks, of shape
+        (batch size, input channels, input rows, input columns).
+        See the optional parameter ``input_shape``.
+
+    filters: symbolic 4D tensor
+        Set of filters used in CNN layer of shape
+        (input channels, output channels, filter rows, filter columns).
+        See the optional parameter ``filter_shape``. **Note: the order for
+        ``output_channels`` and ``input_channels`` is reversed with respect to
+        ``conv2d``.**
+
+    output_shape: tuple/list of len 4 of int or Constant variable
+        The shape of the output of ``conv2d_transpose``.
```

Comment:
I read the docstring before the implementation... since this function simply calls `conv2d_grad_wrt_inputs`, then it is supported.
Could you just update the docstring?